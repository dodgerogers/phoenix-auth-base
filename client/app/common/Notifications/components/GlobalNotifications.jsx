import React from 'react'
import PropTypes from 'prop-types';
import NotificationsContainer from '../containers/NotificationsContainer';
import { areaIDs } from '../constants';
import styled from 'styled-components';

const NotificationWrapper = styled.div`
  position: fixed;
  margin-top: 80px;
  margin-left: -25%;
  max-height: 50%;
  width: 50%;
  left: 50%;
  z-index: 100;
`;

const GlobalNotifications = (props) => {
  return (
    <NotificationWrapper>
      <NotificationsContainer id={areaIDs.APPLICATION} />
    </NotificationWrapper>
  );
}

export default GlobalNotifications;
