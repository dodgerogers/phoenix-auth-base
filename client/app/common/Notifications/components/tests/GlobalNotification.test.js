import React from 'react';
import { shallow } from 'enzyme';
import { fromJS } from 'immutable';
import toJson from 'enzyme-to-json';
import GlobalNotifications from '../GlobalNotifications';


describe('GlobalNotification', () => {
  describe('PureComponent', () => {
    const fullProps = () => ({});

    const pureComponent = (props) => shallow(
      <GlobalNotifications {...props} />
    );

    it('matches snapshot', () => {
      const component = pureComponent(fullProps());

      expect(toJson(component)).toMatchSnapshot();
    });
  });
});
