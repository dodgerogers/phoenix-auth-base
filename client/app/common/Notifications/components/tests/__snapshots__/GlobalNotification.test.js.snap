// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalNotification PureComponent matches snapshot 1`] = `
<styled.div>
  <Connect(NotificationList)
    id="APPLICATION"
  />
</styled.div>
`;
